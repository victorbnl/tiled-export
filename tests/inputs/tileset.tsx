<?xml version="1.0" encoding="UTF-8"?>
<tileset version="1.9" tiledversion="1.9.2" name="tileset" tilewidth="48" tileheight="48" tilecount="3" columns="0">
 <grid orientation="orthogonal" width="1" height="1"/>
 <tile id="2">
  <image width="48" height="48" source="img/dirt.png"/>
 </tile>
 <tile id="0">
  <image width="48" height="48" source="img/grass.png"/>
 </tile>
 <tile id="1">
  <image width="48" height="48" source="img/bricks.png"/>
 </tile>
</tileset>
