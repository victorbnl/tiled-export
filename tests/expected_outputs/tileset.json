{
  "version": "1.9",
  "tiledversion": "1.9.2",
  "name": "tileset",
  "class": "",
  "tilewidth": 48,
  "tileheight": 48,
  "spacing": 0,
  "margin": 0,
  "tilecount": 3,
  "columns": 0,
  "objectalignment": "unspecified",
  "tilerendersize": "tile",
  "fillmode": "stretch",
  "tileoffset": {
    "x": 0,
    "y": 0
  },
  "grid": {
    "orientation": "orthogonal",
    "width": 1,
    "height": 1
  },
  "tiles": [
    {
      "id": 2,
      "image": "img\/dirt.png",
      "imagewidth": 48,
      "imageheight": 48
    },
    {
      "id": 0,
      "image": "img\/grass.png",
      "imagewidth": 48,
      "imageheight": 48
    },
    {
      "id": 1,
      "image": "img\/bricks.png",
      "imagewidth": 48,
      "imageheight": 48
    }
  ]
}
